### ISSUE TITLE
where does the template go to? 

### ISSUE THREAD

**Issue by herrdeh** [quick61/FreeCAD-Drawing-Templates#2]
I couldnt find any hint into which directory the svg file created has to go.

**Comment by quick61** [comment]
They go into the WhereYouInstalledFreeCAD/Mod/Drawing/Templates/ for quickest access to them, but can be anywhere on your system or anywhere your file browser can reach them. 

From the Drawing Workbench, when you click on the Page group that is created when you add a template, you can click on it and in the Data tab, under Templates, you can change the template to whatever you want. The /Mod/Drawing/Templates/ is where your file browser will open to when accessed from there.

**Comment by herrdeh** [comment]
Thank you so much. Would you please put this info into the Wiki?

At my system, when I put a "Drawing" folder into "Mod", after FreeCAD restart the drawing workbench disappears. Properties seem to be allright.

**Comment by herrdeh** [comment]
Well, sorry - that didnt work for me.
According to this:

http://forum.freecadweb.org/viewtopic.php?f=3&t=10904&e=1&view=unread#unread

the templates have to go to /usr/share/freecad/Mod/Drawing/Templates.

I think, the wiki article could be a little broader on that issue.

**Comment by quick61** [comment]
Yes, that would be a location where you installed FreeCAD. (WhereYouInstalledFreeCAD/Mod/Drawing/Templates/). Not knowing what OS, or how you installed FreeCAD lead me to provide the most generic answer. For instance, I compile FreeCAD from source and my release version is installed in /usr/local and my master development version is in /home/me/Build/FreeCAD-test. 

Sorry for any misunderstanding.
